inkcut/device/extensions.py

```diff
@@ -45,23 +56,22 @@ def default_config_view_factory():
 
 class DeviceDriver(Declarative):
     """ Provide meta info about this device """
-    # ID of the device
-    # If none exits one i created from manufacturer.model
+    #: ID of the device. If none exits one is created from manufacturer.model
     id = d_(Unicode())
 
-    # Name of the device (optional)
+    #: Name of the device (optional)
```

Comment:
Does the `:` have meaning here? Some kind of documentation generation tool?

Reply:
Numpy doc uses them for attributes.